### PR TITLE
Attempt to read the DAL values for sound effect editor effect constants

### PR DESCRIPTION
**Pull request by riknoll**
Part of the fix for https://github.com/microsoft/pxt-microbit/issues/5569

Reads the DAL constants for the default effect parameters if possible.

### CODE REVIEW

File: webapp/src/components/soundEffectEditor/soundUtil.ts [hunk 1]
```diff
@@ -289,4 +289,19 @@ export function getMixerGallerySounds(): SoundGalleryItem[] {
...
+    const file = pxt.appTarget.bundledpkgs?.["core"]?.["dal.d.ts"];
+
+    if (!file) return defaultValue;
+
+    const regEx = new RegExp(`${name}\\s*=\\s*(\\d+),`);
+
+    const match = regEx.exec(file);
+
+    if (match) {
+        return parseInt(match[1]);
```
Comment (thsparks) [inline webapp/src/components/soundEffectEditor/soundUtil.ts:304]: Should we protect against non-numeric strings here and return defaultValue?

**Review by thsparks — APPROVED** [review]
One nit, not major

File: webapp/src/components/soundEffectEditor/soundUtil.ts (hunk 1 above)
Comment (thsparks) [reply webapp/src/components/soundEffectEditor/soundUtil.ts:304]: Oh nvm, I guess regex does this already

**Review by thsparks — APPROVED** [review]

File: webapp/src/components/soundEffectEditor/soundUtil.ts (hunk 1 above)
Comment (riknoll) [reply webapp/src/components/soundEffectEditor/soundUtil.ts:304]: yeah, also these are always numbers

**Pull request merged**